src/js/components/awardv2/idv/federalAccounts/FederalAccountsTree.jsx

```diff
@@ -238,7 +238,6 @@ export default class FederalAccountsTree extends React.Component {
         }
         return (
             <div>
-                <h4 id="federal-account-treemap-title">Federal Accounts</h4>
```

Comment:
Can we remove this from the css in `_awardFundingSummary.scss`
`      #federal-account-treemap-title {
        margin-top: 0;
      }`